iox_client_meta_schema.go

```diff
@@ -0,0 +1,110 @@
+package influxdbiox
+
+import (
+	"context"
+	"errors"
+
+	schema "github.com/influxdata/influxdb-iox-client-go/internal/schema"
+)
+
+// ColumnType defines the column data types IOx can represent.
+type ColumnType int32
+
+const (
+	// ColumnTypeUnknown is an invalid column type.
+	ColumnTypeUnknown ColumnType = 0
+	// ColumnType_I64 is an int64.
+	ColumnType_I64 ColumnType = 1
+	// ColumnType_U64 is an uint64.
+	ColumnType_U64 ColumnType = 2
+	// ColumnType_F64 is an float64.
+	ColumnType_F64 ColumnType = 3
+	// ColumnType_BOOL is a bool.
+	ColumnType_BOOL ColumnType = 4
+	// ColumnType_STRING is a string.
+	ColumnType_STRING ColumnType = 5
+	// ColumnType_TIME is a timestamp.
+	ColumnType_TIME ColumnType = 6
+	// ColumnType_TAG is a tag value.
+	ColumnType_TAG ColumnType = 7
+)
```

Comment:
I chose to avoid using the proto-generated consts, to decouple the user of this client away from the wire-format / implementation details.

Reply:
Great idea. We would also like to switch to FlightSQL at some point in the future, so abstractions now is 👍